mapswipe_workers/mapswipe_workers/firebase_to_postgres/transfer_results.py

```diff
@@ -201,4 +208,20 @@ def get_user_ids_from_results(results):
         for userId, results in users.items():
             user_ids.add(userId)
 
-    return user_ids
\ No newline at end of file
+    return user_ids
+
+
+def get_projects_from_postgres():
+    '''
+    Get the id of all projects in postgres
+    '''
+
+    pg_db = auth.postgresDB()
+    sql_query = '''
+        SELECT project_id from projects;
+    '''
+    raw_ids = pg_db.retr_query(sql_query, None)
+    project_ids = [i[0] for i in raw_ids]
+
+    del pg_db
```

Comment:
Is there a particular reason to use `del` here? My understanding is that since `pg_db` is a local variable, it will get deleted at the end of the function, but please correct me if I'm wrong. See [this SO answer](https://stackoverflow.com/a/2193071/1138710) for reference. If it is indeed useful to have the `del` statement here, maybe add a comment to explain why?